src/librustc/middle/traits/select.rs

```diff
@@ -1471,6 +1471,11 @@ impl<'cx, 'tcx> SelectionContext<'cx, 'tcx> {
                     return Ok(ParameterBuiltin);
                 }
 
+                // Whether upvars are sized is checked elsewhere
```

Comment:
Do you mind say *where* they're checked? "Elsewhere" tends to be kind of frustrating when looking at these comments.